pkg/apis/stork/register.go

```diff
@@ -3,6 +3,4 @@ package stork
 const (
 	// GroupName is the group name of the CRD
 	GroupName = "stork.libopenstorage.org"
-	// Version is the version of the CRD
-	Version = "v1alpha1"
```

Comment:
Why remove this?

Reply:
Moved it inside pkg/apis/stork/v1alpha1. It's in the `SchemeGroupVersion` now.

It was better to keep the actual version inside the version package since when we have multiple versions, `pkg/apis/stork/register.go` will be generic.